lib/jekyll.rb

```diff
@@ -78,7 +78,7 @@ class << self
     # images and allows you to skip that when working in development.
 
     def env
-      ENV["JEKYLL_ENV"] || "development"
+      ENV["JEKYLL_ENV"] || "local"
```

Comment:
To me, it's a question of who you want to optimize for. "local" may be more intuitive for non-developer users, but for developers, "local" may be just as unintuitive as "development" is for non-developer users. I'm not strongly opinionated on the matter, but generally suggest not reinventing the wheel wherever possible, even if the wheel isn't 100% ideal.